_sass/_custom.scss

```diff
@@ -0,0 +1,108 @@
+.center {
+  text-align: center;
+}
+
+h2.center {
+  @extend .center;
+  margin-top: 80px;
+}
+
+span.left-pad {
+  padding-left: 20px;
+}
+
+.email-form {
+  input {
+    display: block;
+    border-radius: 5px;
+  }
+  input#bd-email {
+    margin: 20px auto;
+    @include media($medium) {
+      width: 75%;
+    }
+    // border-color: $primary-color;
+  }
+  input[type="submit"] {
+    text-align: right;
+    margin: 0px auto;
+    background-color: $primary-color;
+    color: $primary-white;
+  }
+}
+
+div.post-entry-title {
+  @include font-size(18);
+  color: $primary-color;
+  margin-top: 16px;
+  text-transform: capitalize;
+
+  @include media($medium) {
+    @include font-size(22);
+    margin-bottom: 10px;
+  }
+}
+
+.empty-title {
+   display: block;
+   height: 40px;
+ }
+
+#test {
+  margin-top: 60px;
+}
+
+.btn-demo {
+  margin: 15px;
+}
+
+p code, li code {
+  color: #000;
+}
+
+.logo-container {
+  width: 100%;
+}
+
+@media (max-width: 600px) {
+  .third {
+    width: 80%;
+    margin: 0px 10%;
+  }
+}
+
+@media (min-width: 601px) {
+  .third {
+    width: 32.5%;
+  }
+}
+
+.third {
+  padding: 0px 15px;
+  display: inline-block;
+  text-align: center;
+  img {
+    max-height: 100px;
+    margin: 30px auto;
+    vertical-align: middle;
+  }
+}
+
+.btn.help {
+  display: block;
+  margin: 0px auto;
+  text-align: center;
+  padding: 20px 40px;
+  margin: 20px;
+  font-size: 20px;
+}
+
+.btn {
+  a, a:visited {
+    color: $secondary-color;
```

Comment:
I would presume my comment at https://github.com/foss-responders/fossresponders.com/pull/1#issuecomment-598574053 is related to something here. I think the sass ends up setting a specific colour on both the anchor and the containing button div. So, the div:hover only affects the button, leaving the override on the anchor to change only when we reach a:hover

Reply:
I don't think this is nitpicking. I should have fixed this at the time, but was blanking as to how. I fixed it using `span`.